fe/fe-core/src/test/java/com/starrocks/plugin/PluginZipTest.java

```diff
@@ -134,8 +135,8 @@ public void testDownloadAndValidateZipIOException() {
     @Test
     public void testExtract() {
         try {
-            Files.copy(PluginTestUtil.getTestPath("source/test.zip"), PluginTestUtil.getTestPath("source/test-a.zip"));
-
+            Files.copy(PluginTestUtil.getTestPath("source/test.zip"), PluginTestUtil.getTestPath("source/test-a.zip"),
+                    StandardCopyOption.REPLACE_EXISTING);
```

Comment:
why add this code?

Reply:
if not add REPLACE_EXISTING option, the second time the unit test is executed, it will fail